tests/integration_tests/performance/test_network_ab.py

```diff
@@ -96,10 +96,10 @@ def test_network_latency(network_microvm, metrics, iteration):
         assert rc == 0, stderr
 
         samples.extend(consume_ping_output(ping_output))
+    fcmetrics.stop()
```

Comment:
It is not clear to me from the commit message why this change fixes the CI issue. I'm probably missing something, but could you try to explain it a bit better?

Reply:
Unfortunately, since the issue was intermittent, it was very hard to reproduce and debug so, I was not able to find a rational explanation why this fixed the issue. I changed the sequence because I was working with metrics and that is the only difference I saw in the ci framework at that time; fcmetrics was a new thread added to the ci so I suspected it could have some kind of affect on the test but from the multiple tests I tried the thread didn't seem to cause any issue but after the change in sequence I didn't see the issue in the tests so I thought it would be useful to add it.
Worst case scenario is that the issue is still there and I was just not able to reproduce it, in this case I think its still a CI issue and not Firecracker so I feel the change to rearrange the sequence should be ok to have.